gradle-plugin/settings.gradle

```diff
@@ -1 +1,46 @@
-rootProject.name = "gradle-plugin"
\ No newline at end of file
+pluginManagement {
+  repositories {
+    google()
+    gradlePluginPortal()
+    mavenCentral()
+  }
+}
+
+dependencyResolutionManagement {
+  versionCatalogs {
+    libs {
+      // TODO remove this once the dirs aren't symlinked anymore
```

Comment:
Does this mean remove the commented out `from(files` line, the `if (System.getenv("DEP_OVERRIDES") == "true")` block, or something else? Could use a little more explicitness on the comment. 

Also, are we not able to remove the symlinks now?

Reply:
With version catalogs you could actually start using Renovate and let it handle updates like that for you. Gradle in general really doesn't like symlinks. I can leave it in place and remove the comment if you want though